handbook/engineering/search/goals.md

```diff
@@ -4,48 +4,48 @@
-      - Return results faster for large result sets, e.g. indexed repositories in very large codebases.
-  - **In progress work:** [Streaming search](#5-streaming-search)
+TODO table of use cases / capabilities we want to achieve, comparing Sourcegraph with other search tools
 
-### Expressive
+**Planned work**
 
-- **Extend Search Query Language**
-  - **Problem:** It is imperative that users can find and filter the code, files, repositories, and commits they care about in extremely large codebases.
-  - **Outcomes:** Users can express relations on code, files, repositories, and commits in search queries to more effectively filter the data they need. Sourcegraph is the exclusive industrial-strength search solution that provides these capabilities.
-  - **Planned work:** [Search language rules engine](#10-search-language-rules-engine)
+1. Search expressions
+1. Search language rules engine
 
-### Easy to use
+### Performant search users can trust
 
-- **Improve search experience**
-  - **Problem:**
-      - New users who are trying Sourcegraph for the first time have trouble learning the syntax and breadth of Sourcegraph features.
-          - **In progress work:** [Search tour](#2-search-tour)
-      - It is hard for users to quickly get to code they care about.
-          - **In progress work:** [Enterprise homepage](#1-enterprise-homepage)
-  - **Outcomes:**
-     - New users introduced to Sourcegraph are able to quickly run searches that show them the value of Sourcegraph.
-     - Users can run searches over code they care about more quickly.
-     - Sourcegraph surfaces code and searches users care about.
+**Problem:** Search is too slow or frequently times out at the scale some of our customers care about, and at the scale we want Sourcegraph Cloud to reach. In large-scale codebases, running the same search several times yields varying result counts, with no insights into the underlying reasons. Customers cannot trust that their codebase was exhaustively searched.
 
-### Shareable
+**Success criteria**
 
-- **Code monitoring** ([RFC 227](https://docs.google.com/document/d/1_R5DgpUkxyZilsJ9vBQm5cvRPT2udc3tZIPg2q3cnZU/edit))
-  - **Problem:** Users want to be notified about important things going on in their code.
-  - **Outcome:** Notifications create a shared understanding and raise awareness of what’s going on in the code.
-  - **Planned work:** [Code monitoring version 1](#6-code-monitoring-version-1)
+- **By EOY 2020**: Latency for the set of search queries run by search-blitz is less than TODO Nms with 500k repositories indexed on Sourcegraph Cloud
+    - 2020-11-05: TODO
+- **By EOY 2020**: P99 time to first result is less than TODO Nms across all search types on Sourcegraph Cloud
+    - 2020-11-05: TODO
+
+
+**Planned work**
+
+1. Scale indexed text search to 500k repositories
+1. Streaming search
+
+### Sourcegraph helps customers monitor important changes to their codeebase
```

Comment:
Could build on this to make it a bit more grand and a larger vision:

```suggestion
### Sourcegraph is part of the development workflow and proactively notifies of important code changes
```

Reply:
I do like the "proactive" part, I'm less sold on the "part of the development workflow". Cutting the apple in half and at least fixing the typo :)

```suggestion
### Sourcegraph proactively notifies of important code changes
```